models/chatroom.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type ChatRoom struct {
```

Comment:
please add comments for exported structs

Reply:
Added.